src/site/_includes/pattern.njk

```diff
@@ -1,7 +1,9 @@
 ---
-layout: patterns
+layout: pattern-page
 pageScripts:
   - '/js/content.js'
+inlineScripts:
+   - '../../../lib/components/Tabs/index.js'
```

Comment:
It looks like this causes a runtime error because the inline script is trying to use module imports but we're inlining it using a regular script tag. Given how many dependencies the Tabs widget has, maybe inlining it isn't the right approach. It seems like we'll need to use placeholder styles to prevent CLS and hide the content until it's ready.